azure-quantum/azure/quantum/qiskit/job.py

```diff
@@ -197,6 +198,23 @@ def _format_ionq_results(self, sampler_seed=None):
 
         return {"counts": counts, "probabilities": probabilities}
 
+    @staticmethod
+    def _azure_to_qiskit(obj):

```

Comment:
I would name this something like `_to_qiskit_bitstring`, as this converts the result key from the QIR format to one supported by Qiskit.

Reply:
what do you think of my counter proposal?